tests/unit/test_charm.py

```diff
@@ -116,35 +117,65 @@ def test_remote_write_configuration(self, mock_push: MagicMock):
         )
 
         path, content = mock_push.call_args[0]
-        self.assertEqual(path, "/etc/agent/agent.yaml")
-        self.assertDictEqual(
-            yaml.load(content, Loader=yaml.FullLoader),
-            {
-                "integrations": {
-                    "agent": {
-                        "enabled": True,
-                        "relabel_configs": REWRITE_CONFIGS,
-                    },
-                    "prometheus_remote_write": [
-                        {"url": "http://1.1.1.2:9090/api/v1/write"},
-                        {"url": "http://1.1.1.1:9090/api/v1/write"},
-                    ],
-                },
-                "prometheus": {
-                    "configs": [
-                        {
-                            "name": "agent_scraper",
-                            "remote_write": [
-                                {"url": "http://1.1.1.2:9090/api/v1/write"},
-                                {"url": "http://1.1.1.1:9090/api/v1/write"},
-                            ],
-                            "scrape_configs": [],
-                        }
-                    ]
+        content = yaml.load(content, Loader=yaml.FullLoader)
```

Comment:
why not `yaml.safe_load`?

Reply:
I didn't modify the way the dictionary was loaded, I only assign it to a variable.
I'll change it.